batch/batch_network_analysis/algorithm.py

```diff
@@ -1,37 +0,0 @@
-import plot
```

Comment:
what happened to this file?

Reply:
that file still in the aws/batch/batch_network_analysis folder
![image](https://github.com/ncsa/standalone-smm-analytics/assets/12547573/0168bcac-9936-4fe8-8032-14adfda098ac)